sqlx-core/src/any/options.rs

```diff
@@ -47,7 +48,67 @@ impl AnyConnectOptions {
     }
 }
 
-#[derive(Debug)]
+#[cfg(feature = "postgres")]
+impl TryFrom<AnyConnectOptions> for PgConnectOptions {
+    type Error = Error;
+
+    fn try_from(value: AnyConnectOptions) -> Result<Self, Self::Error> {
+        if let AnyConnectOptionsKind::Postgres(connect_options) = value.0 {
```

Comment:
Can we fix all these warnings? It should be as simple as `#[allow(irrefutable_if_let)]` or whatever the lint actually is.

Reply:
Done